tibco_ems/datadog_checks/tibco_ems/tibco_ems.py

```diff
@@ -0,0 +1,247 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+from typing import Any  # noqa: F401
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.subprocess_output import get_subprocess_output
+
+from .constants import SHOW_METRIC_DATA, unit_pattern
+
+DEFAULT_HOST = 'localhost'
+DEFAULT_PORT = 7222
+TO_BYTES = {'B': 1, 'Kb': 1e3, 'Mb': 1e6, 'Gb': 1e9, 'Tb': 1e12}
+CONNECTION_STRING = 'tcp://{}:{}'
+
+
+class TibcoEMSCheck(AgentCheck):
+
+    # This will be the prefix of every metric and service check the integration sends
+    __NAMESPACE__ = 'tibco_ems'
+
+    def __init__(self, name, init_config, instances):
+        super(TibcoEMSCheck, self).__init__(name, init_config, instances)
+        # Allow to specify a complete command for tibemsadmin such as `docker exec <container> tibemsadmin`
+        default_tibemsadmin_cmd = init_config.get('tibemsadmin', 'tibemsadmin')
```

Comment:
```suggestion
        default_tibemsadmin_cmd = init_config.get('tibemsadmin', '/usr/bin/tibemsadmin')
```

This should match the default provided in the config.